crates/common/src/macros.rs

```diff
@@ -309,3 +377,46 @@ macro_rules! felt_bytes {
         }
     }};
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn macro_felt_example() {
+        let single: pathfinder_crypto::Felt = felt2!("0xABCD");
+        let vector: Vec<pathfinder_crypto::Felt> = felt2!["0x1", "0x2", "0x4xxx"];
+
+        let tx = crate::transaction::InvokeTransactionV3 {
+            signature: felt2!["0xAAAA", "0xBBBB"],
```

Comment:
Personally I think this looks strange. It's very implicity that the result will be a Vec here.